spec/timezone.html

```diff
@@ -3,5 +3,240 @@
 
 <emu-clause id="sec-temporal-timezone">
   <h1>Temporal.TimeZone Objects</h1>
-  <p><mark>TODO: Specify.</mark></p>
+  <p>A Temporal.TimeZone object is an immutable Object referencing a time zone.</p>
+
+  <emu-clause id="sec-temporal-timezone-constructor">
+    <h1>The Temporal.TimeZone Constructor</h1>
+    <p>
+      The Temporal.TimeZone constructor is the %Temporal.TimeZone% intrinsic object.
+      When called as a constructor, it creates and initializes a new Temporal.TimeZone object.
+    </p>
+    <p>
+      The Temporal.TimeZone constructor is designed to be subclassable.
+      It may be used as the value of an extends clause of a class definition.
+      Subclass constructors that intend to inherit the specified TimeZone behaviour must include a super call to the %Temporal.TimeZone% constructor to create and initialize subclass instances with the necessary internal slots.
+    </p>
+
+    <emu-clause id="sec-temporal.timezone">
+      <h1>Temporal.TimeZone ( _identifier_ )</h1>
+      <p>
+        When the `Temporal.TimeZone` function is called, the following steps are taken:
+      </p>
+      <emu-alg>
+        1. If NewTarget is *undefined*, then
+          1. Throw a *TypeError* exception.
+        1. Let _identifier_ be ? ToString(_identifier_).
+        1. If _identifier_ matches `/([+-][01]?[0-9]):?([0-5][0-9])/`, then
```

Comment:
We don’t have any precedent for this kind of comparison (regex notation); I’d expect either making an actual regex and calling abstract ops to test it, or, a combination of string ops and prose.

Reply:
+1

Reply:
+1